index.html

```diff
@@ -851,6 +851,7 @@ <h3>Landmark Roles</h3>
 		<section id="live_region_roles">
 			<h3>Live Region Roles</h3>
 			<p>The following <a>roles</a> are <a>live regions</a> and can be modified by <a href="#attrs_liveregions">live region attributes</a>.</p>
+			<p>Typically, assistive technology only speaks <em>changes</em> to a live region, not the initial contents of a live region. To ensure content in a live region is announced, authors SHOULD create an rendered but empty live region as early as possible (such as on page load), and then add content to the live region when the author wishes it to be announced. The exception to this live region convention is <code>alert</code>, due to system accessiblity notifications events required for the role. While an <rref>alert</rref> is a live region, its content is announced by assistive technology when the alert is rendered on the page and when the content changes.</p>
```

Comment:
nit: "an rendered" -> "a rendered"

Reply:
There's an implicit assumption in this paragraph about aria-relevant, because it says authors should "add content to the live region when the author wishes it to be announced", but maybe we could clarify slightly to make sure that it's clear that changes and removals would be announced too.

Reply:
Yeah, I wondered if that would come up in review. "modify the content" vs "add content"?

Reply:
```suggestion
			<p>Typically, assistive technology only speaks <em>changes</em> to a live region, not the initial contents of a live region. To ensure content in a live region is announced, authors SHOULD create a rendered but empty live region as early as possible (such as on page load), and then modify the content of the live region when the author wishes changes to be announced. The exception to this live region convention is <code>alert</code>, due to system accessibility notifications events required for the role. While an <rref>alert</rref> is a live region, its content is announced by assistive technology when the alert is rendered on the page and when the content changes.</p>
```